Mirror/Runtime/Transport/EpicOnlineTransport/EOSSDKComponent.cs

```diff
@@ -412,21 +452,25 @@ public class EOSSDKComponent : MonoBehaviour {
         }
 
         private void OnApplicationQuit() {
-            if (EOS != null) {
+#if !UNITY_EDITOR
```

Comment:
Nitpick: the block after this has #if UNITY_EDITOR while this is the inverse of that test. In this case I would move this block to the #else case of the if. This would only give you one affirmative test (no ! in the test) which is slightly easier to read.